### PR TITLE
Fix failing roundtrip encrypt/decryptObject tests.

### PR DESCRIPTION
**Pull request by mandyvenables**
Adding a failing test, needs to be debugged and pass.

### CODE REVIEW

File: algorithms/aeskw.js [hunk 1]
```diff
@@ -45,16 +45,16 @@ class Kek {
   async unwrapKey({wrappedKey}) {
     const kek = this.key;
     // Note: `AES-GCM` algorithm name doesn't matter; will be exported raw.
-    wrappedKey = base64url.decode(wrappedKey);
+    wrappedKey = new Uint8Array(base64url.decode(wrappedKey));
```
Comment (mattcollier) [inline algorithms/aeskw.js]: It appears to me that the return value of `base64url.decode` is already a Uint8Array.

https://github.com/digitalbazaar/base64url-universal/blob/master/lib/base64url.js#L87-L109

File: algorithms/x25519.js [hunk 2]
```diff
@@ -49,8 +53,18 @@ export async function kekFromEphemeralPeer({keyAgreementKey, epk}) {
...
-// static key
+/**
+ * (Encryption case) Gets Kek *and* ephemeral DH key from a peer's public
+ * static key.
+ *
+ * @param {object} options - Options hashmap.
+ * @param {object} options.ephemeralKeyPair - Ephemeral key pair.
+ * @param {object} options.staticPublicKey - Static public key.
+ * @typedef {object} Kek
+ * @returns {Promise<{
```
Comment (mattcollier) [inline algorithms/x25519.js]: linting issue here.

**Review by mattcollier — APPROVED** [review]
TY, syntax looks good.  One issue regarding UInt8Array.

File: algorithms/aeskw.js (hunk 1 above)
Comment (JSAssassin) [reply algorithms/aeskw.js]: You're right, I've removed it here 3dd73ee. Thanks!

File: algorithms/x25519.js (hunk 2 above)
Comment (JSAssassin) [reply algorithms/x25519.js]: Fixed here eb716c6

File: algorithms/x25519.js [hunk 3]
```diff
@@ -61,8 +76,9 @@ export async function kekFromStaticPeer({ephemeralKeyPair, staticPublicKey}) {
     throw new Error(
       `"staticPublicKey.type" must be "${KEY_TYPE}".`);
   }
-  const publicKeyBase58 = staticPublicKey.publicKeyMultibase.slice(1);
-  const remotePublicKey = base58.decode(publicKeyBase58);
+  const remotePublicKey = multibaseDecode(
+    MULTICODEC_X25519_PUB_HEADER, staticPublicKey.publicKeyMultibase
+  );
```
Comment (dlongley) [inline algorithms/x25519.js]:
```suggestion
  const remotePublicKey = multibaseDecode(
    MULTICODEC_X25519_PUB_HEADER, staticPublicKey.publicKeyMultibase);
```

File: algorithms/x25519.js [hunk 4]
```diff
@@ -49,8 +53,19 @@ export async function kekFromEphemeralPeer({keyAgreementKey, epk}) {
   };
 }
 
-// Encryption case: get Kek *and* ephemeral DH key from a peer's public
-// static key
+/**
+ * (Encryption case) Gets Kek *and* ephemeral DH key from a peer's public
+ * static key.
```
Comment (dlongley) [inline algorithms/x25519.js]:
```suggestion
 * (Encryption case) Generates KEK from ephemeral DH private key and a peer's public
 * static key.
```

**Review by dlongley — APPROVED** [review]
Thanks! See nits.

File: algorithms/x25519.js (hunk 3 above)
Comment (dmitrizagidulin) [reply algorithms/x25519.js]: @dlongley - I thought our style guide preferred the former?

**Review by dmitrizagidulin — APPROVED** [review]
:+1:

File: algorithms/x25519.js (hunk 3 above)
Comment (dlongley) [reply algorithms/x25519.js]: Nope, and we've got reams and reams of code where the only thing you drop are the params. You only drop the parenthesis if you've got a multiline object that needs to be closed, eg:

```js
foo({
  bar: 'baz'
});
```

File: algorithms/x25519.js (hunk 4 above)
Comment (JSAssassin) [reply algorithms/x25519.js]: Updated d254eb8

File: algorithms/x25519.js (hunk 3 above)
Comment (JSAssassin) [reply algorithms/x25519.js]: Updated here d254eb8

**Comment by dmitrizagidulin** [comment]
@mandyvenables - Package `@digitalbazaar/minimal-cipher@4.0.0` published; this should unblock you in the other PR.

**Pull request merged**